draco/analysis/beamform.py

```diff
@@ -1199,6 +1208,13 @@ def setup(self, hpmap: containers.Map):
         """
 
         self.map = hpmap
+        mv = self.map.map[:]
+
+        self.log.info("Smoothing input Healpix map.")
+        for lfi, _ in mv.enumerate(axis=0):
+            mv[lfi, 0] = healpy.smoothing(
```

Comment:
Do we also want to loop over the pol axis here instead of just smoothing the zeroth pol, since in line 1273 we form beams for each pol present in the map container?

Reply:
Yes. Not quite clear why it doesn't do that already.

Reply:
This should be fixed now.